.github/workflows/deploy.yml

```diff
@@ -0,0 +1,16 @@
+name: Fly Deploy
+on:
+  push:
+    branches:
+      - master
```

Comment:
Maybe we should have it set to listen to the `deployment` event, since I'm still not convinced that this will consistently execute after a build.

Reply:
There's no ui for the deployment events. I think it's more intended to be a jenkins plugin or something.
I _believe_ this should fire after we merge this pr.